src/com/nutrons/framework/producers/Serial.java

```diff
@@ -0,0 +1,52 @@
+package com.nutrons.framework.producers;
+
+import static com.nutrons.framework.util.FlowOperators.toFlow;
+import edu.wpi.first.wpilibj.SerialPort;
+
+import io.reactivex.Flowable;
+
+/**
+ * A wrapper around WPI's SerialE class which provides Flowables for data sent to the roboRIO over serial
+ */
+public class Serial {
+    private final SerialPort serial;
+    private final int bufferSize;
+    private final int packetLength;
+    private final Flowable<byte[]> dataStream;
+    private final char terminationCharacter;
+
+    public Serial(int bufferSize, int packetLength, char terminationCharacter) {
+        this(9600, SerialPort.Port.kUSB, bufferSize, packetLength, '\n');
```

Comment:
The `terminationCharacter` parameter is unused.  You have a convenience constructor which sets it to a default